crates/sui-sdk/src/json_rpc_error.rs

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+use jsonrpsee::types::{error::UNKNOWN_ERROR_CODE, ErrorObjectOwned};
```

Comment:
Doesn't this line imply that users of the SDK still need to depend on the jsonrpsee crate to interpret errors?  It should be trivial to get rid of the reference to `UNKNOWN_ERROR_CODE` but not so sure about `ErrorObjectOwned`.